app/views/welcome/index.html.erb

```diff
@@ -43,3 +109,10 @@
     <li>Tran Le - Engineer</li>
   </ul>
 </section>
+</div>
+<style media="screen">
+  body {
+    background-image: url('/assets/images/background.jpg');
```

Comment:
Since this is CSS, not an image tag, you should use the [image path helper](https://api.rubyonrails.org/classes/ActionView/Helpers/AssetUrlHelper.html#method-i-image_path) rather than an asset tag.